extensions/database/module/langs/translation-fr.json

```diff
@@ -15,15 +15,15 @@
     "database-source/alert-invalid-query-select": "La requête doit débuter par le mot-clé SELECT",
     "database-source/form-validation-failure": "Le nouveau formulaire de connexion est invalide !",
     "database-source/alert-connection-edit": "La connexion a été éditée avec succès",
-    "database-source/connectionNameLabel": "Nom :",
-    "database-source/databaseTypeLabel": "Type :",
-    "database-source/databaseHostLabel": "Hôte :",
-    "database-source/databasePortLabel": "Port :",
-    "database-source/databaseUserLabel": "Utilisateur :",
-    "database-source/databasePasswordLabel": "Mot de passe :",
-    "database-source/databaseNameLabel": "Base de données :",
-    "database-source/databaseFileNameLabel": "Fichier de données :",
-    "database-source/databaseSchemaLabel": "Schéma :",
+    "database-source/connectionNameLabel": "Nom ",
+    "database-source/databaseTypeLabel": "Type ",
```

Comment:
I guess we probably also want to remove the trailing whitespace in those strings.